birdhouse/env.local.example

```diff
@@ -242,7 +242,8 @@ export GEOSERVER_ADMIN_PASSWORD=geoserverpass
 # Jupyter single-user server images
 #export DOCKER_NOTEBOOK_IMAGES="pavics/workflow-tests:210216 \
 #                               pavics/crim-jupyter-eo:0.3.0 \
-#                               pavics/crim-jupyter-nlp:0.4.0"
+#                               pavics/crim-jupyter-nlp:0.4.0 \
+#                               birdhouse/pavics-jupyter-base:mlflow-proxy"
```

Comment:
Just curious, why is this the base image?   I thought on CRIM's side, all specific image are child image of the base image and the base image stays generic?

Reply:
It is different in this case because the introduction of the MLflow extension breaks the Google Drive extension, since the Jupyter version is bumped, and gdrive is not compatible with latest changes (Jupyter v1 or v2 only, project archived and not updated since 2020). 

In the long run, we will probably bump the other images as well to realign them all with the same base image at some point, but this allows temporarily to keep the older Jupyter v1 images and this updated v4 one at the same time.

Reply:
Not sure I follow here.  JupyterHub upgrade from v1 to v4 is different from this new Jupyter image for mlflow-proxy.

My original question that I expected a child mlflow image from the base image, not a direct mlflow flavor from the base image.

I was also curious if both changes have to happen in lock step.

Reply:
The base image use `jupyterlab==2.2.9` which does not respect the requirements for the mlflow-proxy plugin. But upgrading the base image and all the dependencies would cause compatibility issues for the two other child images.(Which would take more time to investigate and resolve)

The safest and most efficient choice is to have a spin off of the base image. If there is a need to update nlp and eo images, we would just need to update the env.local.example. 

JupyterHub upgrade from v1 to v4 is needed to run this new image. Since there is a compatibilty issue with `jupyter-server 2.7.3` and Jupyterhub v1. (when using `Spawner.disable_user_config = True` )